src/components/Paginate.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+
+import { Pagination, PaginationItem, PaginationLink, Collapse } from 'reactstrap';
+
+class Paginate extends React.Component {
```

Comment:
This could just be a function component; doesn't need to be a full class.

Reply:
Sorry, I don't understand, how should I change it?

Reply:
```jsx
const Paginate = ({pages}) => {
  return (
    <html stuff>
  )
}